openslides_backend/services/vote/adapter.py

```diff
@@ -23,6 +23,9 @@ def retrieve(self, endpoint: str, payload: Optional[Dict[str, Any]] = None) -> A
         message = f"Vote service sends HTTP {response.status_code} with the following content: {str(response.content)}."
         if response.status_code < 400:
             self.logger.debug(message)
+        elif response.status_code == 500:
+            self.logger.error(message)
+            raise VoteServiceException("Vote service sends HTTP 500.")
```

Comment:
Change message to "Vote service sends HTTP 500 Internal Server Error."

Reply:
Done.